shared/params/config.go

```diff
@@ -231,8 +230,6 @@ func DemoBeaconConfig() *BeaconChainConfig {
 	demoConfig.MaxEffectiveBalance = 32 * 1e9
 	demoConfig.EjectionBalance = 1.6 * 1e9
 	demoConfig.EffectiveBalanceIncrement = 0.1 * 1e9
-	demoConfig.SyncPollingInterval = 1 * 10 // Query nodes over the network every slot.
-	demoConfig.MinGenesisTime = 0
```

Comment:
Note: this wasn't an override so I removed it. Minimal has min genesis time of 0 too.